oic.r.sp.raml

```diff
@@ -0,0 +1,67 @@
+#%RAML 0.8
+title: Security Profile Resource
+version: v1.0-20180508
+documentation:
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+schemas:
+  - SP: !include schemas/oic.r.sp.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/sp:
+  description: |
+    Resource specifying supported and active security profile(s)
+
+  displayName: Security Profile Resource
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves Security Profile data
+    responses:
+      200:
+        body:
+          application/json:
+            schema: SP
+            example: |
+              {
+                "supported_profiles" : ["oic.sec.sp.baseline", "oic.sec.sp.black"],
+                "active_profile" : "oic.sec.sp.baseline",
+                "credid" : 1
+              }
+      400:
+        description: The request is invalid.
+
+  post:
+    description: |
+      Sets or updates device provisioning status data.
+    body:
+      application/json:
+        schema: SP
+        example: |
+          {
+            "supported_profiles" : ["oic.sec.sp.baseline", "oic.sec.sp.black"],
```

Comment:
if suppported_profiles is readOnly then the schema should be split out for get and post.
this is done since now the readOnly property should not be required in the post.

Reply:
suppported_profiles is RW. So this should be ok.